mypy/stubtest.py

```diff
@@ -1628,7 +1636,7 @@ def anytype() -> mypy.types.AnyType:
                 arg_names.append(
                     None if arg.kind == inspect.Parameter.POSITIONAL_ONLY else arg.name
                 )
-                has_default = arg.default == inspect.Parameter.empty
+                has_default = arg.default is inspect.Parameter.empty
```

Comment:
`has_default` here is a misnomer, it is actually a `no_default`

Reply:
Could you rename the variable while we're here?